### DISCUSSION TITLE
Redesign - Login, file and setting screen

### DISCUSSION THREAD

**Discussion by edwardally** [elementary/triage#381] [Ideas]
No offense intended, but I feel that the login, file and settings GUI is outdated and old-fashioned. Personally, I believe EO8 should undergo an overhaul to improve the visual effects and interface.

**Comment by teamcons** [comment]
what do you call outdated ? Or old-fashioned ?
Do you have examples of where you think it should go, and why ?
What do you call improvement ?
How would it improve usability ?

**Reply by edwardally** [reply]
As mentioned earlier, this is solely my personal perspective, offered without any intent to cause offense. I recognize that not everyone may share the same viewpoint. Despite not being a designer or artist, I'm sharing my thoughts on the design, although providing suggestions on how to enhance or define its ideal look is a challenge for me.

P/S
I've been a happy user since EOS 5 until now and continue.. cheers

**Reply by teamcons** [reply]
"just make it better" unfortunately says very little, which is why i ask you to elaborate.
Nobody knows what you think if you do not share it.

**Comment by edmundlaugasson** [comment]
You can consider to use [lightdm-settings](https://github.com/linuxmint/lightdm-settings) from Linux Mint as this is also based on Ubuntu LTS, like Elementary OS.

**Reply by lenemter** [reply]
elementary OS uses custom made lightdm frontend. I don't think lightdm-settings will work.

**Comment by danirabbit** [comment]
Gonna close this since it’s too vague and doesn’t really offer any suggestions or ideas. Plus for OS 8 we did make big design changes for Settings and the Lock Screen. Feel free to open a new discussion post with more specific ideas or suggestions though in the future :)
